### PR TITLE
Add multiple file metadata updates

### PR DESCRIPTION
**Pull request by MancunianSam**
Create a new multiple file metadata endpoint.
    
The existing updateBulkFileMetadata adds the same metadata values to multiple files. For the new backend checks, we need a way of adding different property values (ServerSideChecksum) to multiple files.
    
I've added this in and updated the tests to point to the new one. It's only the api update lambda that uses this endpoint so once this is all deployed, we can remove the old one.

### CODE REVIEW

File: schema.graphql [hunk 1]
```diff
@@ -19,6 +19,10 @@ input AddFileAndMetadataInput {
 }
 
 input AddFileMetadataWithFileIdInput {
+  metadataInputValues: [AddFileMetadataWithFileIdInputValues!]!
+}
+
+input AddFileMetadataWithFileIdInputValues {
```
Comment (TomJKing) [inline schema.graphql:25]: Maybe rename this `ÀddFileMetadataInputValues` the "FileId" seems a bit superfluous.

File: src/main/scala/uk/gov/nationalarchives/tdr/api/graphql/fields/FileMetadataFields.scala [hunk 2]
```diff
@@ -24,7 +24,9 @@ object FileMetadataFields {
 
   case class BulkFileMetadata(fileIds: Seq[UUID], metadataProperties: Seq[FileMetadata])
 
-  case class AddFileMetadataWithFileIdInput(filePropertyName: String, fileId: UUID, value: String) extends FileMetadataBase
+  case class AddFileMetadataWithFileIdInput(metadataInputValues: List[AddFileMetadataWithFileIdInputValues])
+
+  case class AddFileMetadataWithFileIdInputValues(filePropertyName: String, fileId: UUID, value: String) extends FileMetadataBase
```
Comment (TomJKing) [inline src/main/scala/uk/gov/nationalarchives/tdr/api/graphql/fields/FileMetadataFields.scala:29]: See comment in schema about renaming this.

**Review by TomJKing — COMMENTED** [review]
Looks good. Just a couple of minor comments.

File: schema.graphql (hunk 1 above)
Comment (MancunianSam) [reply schema.graphql:25]: I've updated the case class which will update this input as well. I think this name change will break the API update so we'll have to think about how to deploy this.

**Review by TomJKing — APPROVED** [review]
Looks good. Thanks

**Pull request merged**